src/backend/distributed/sql/udfs/citus_shards/latest.sql

```diff
@@ -28,6 +28,15 @@ ON
     pg_dist_shard.shardid = shard_sizes.shard_id
 WHERE
    pg_dist_placement.shardstate = 1
+AND
```

Comment:
1 line comment would be nice as it might not be very trivial for readers to follow